packages/grafana-ui/src/components/FileDropzone/FileDropzone.tsx

```diff
@@ -134,9 +135,15 @@ export function FileDropzone({ options, children, readAs, onLoad, fileListRender
   const removeFile = (file: DropzoneFile) => {
     const newFiles = files.filter((f) => file.id !== f.id);
     setFiles(newFiles);
+    onFileRemove && onFileRemove(file);
```

Comment:
```suggestion
    onFileRemove?.(file);
```
This might be a bit cleaner.